OneTrustConsentFilter/lib/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.4'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
     compileOnly 'com.onetrust.cmp:native-sdk:202212.1.0.0'
-    compileOnly 'com.rudderstack.android.sdk:core:1.9.0-SNAPSHOT'
+    compileOnly 'com.rudderstack.android.sdk:core:1.8.1-SNAPSHOT'
```

Comment:
does this have to be a snapshot version? I would expect a release one

Reply:
Till rudderstack sdk is released, this is the one with Consent filter capabilities.